### ISSUE TITLE
Wrong line number on chrome sourcMapping

### ISSUE THREAD

**Issue by lucascosta** [jasonsanjose/brackets-sass#88]
Line numbers on Chrome always show the last line in the scss. Example:
```css
html {
	background: red;
}
body {
	background: blue;
}
```
Both the html and body background properties are mapped to **theme.scss:5**

My `.brackets.json`

```json
{
	"sass.enabled": true,
	"sass.options": {
		"outputDir": "../css/",
		"sourceComments": false,
		"outputStyle": "nested"
	}
}
```

I'm on Windows 8.

**Comment by adamsdally** [comment]
I had a similar thing happen to me, which was extremely confusing considering how it was the first time I've ever used Sass. So I found a different sass compiler (Koala) and it works just fine. Here are the differences between the source mappings:
brackets-sass:
    {
      "version": 3,
      "file": "test.css",
      "sources": [
        "test.scss"
      ],
      "sourcesContent": [],
      "mappings": "AAAA,AACI,AAAQ,AAEZ,AACI,AAAU",
      "names": [],
      "sourceRoot": ""
    }

And Koala's output:
    {
    "version": 3,
    "mappings": "AAAA,IAAK;EACD,MAAM,EAAE,KAAK;;AAEjB,CAAE;EACE,SAAS,EAAC,IAAI",
    "sources": ["test.scss"],
    "names": [],
    "file": "test.css"
    }

**Comment by jasonsanjose** [comment]
@lucascosta @adamsdally Looks like this is fixed in the latest dev build https://github.com/jasonsanjose/brackets-sass/releases/tag/2.0.1-105. Probably due to a libsass update. If you can, please follow the instructions for testing the dev build and let me know how it goes.

**Comment by jasonsanjose** [comment]
Fix in #100. Closing.

**Comment by adamsdally** [comment]
This does appear to have fixed the issue. However, I am only able to view the connected stylesheets in Chrome and not Firefox. In Firefox I am getting a "style sheet could not be loaded" error on the scss file. 

I recompiled again with Koala and the source maps are practically identical with the except of the mappings and the line 
    
    "sourcesContent": [],

does not exist in the Koala map file.
